custom_helpers.go

```diff
@@ -1,5 +1,22 @@
 package gotgbot
 
+import (
+	"fmt"
+	"strings"
+)
+
+/* GetLink is a helper method to easily get the message link excluding group and private chat types.
+(It will return en empty string in case of private or group chat type)*/
+func (m Message) GetLink() string {
+	if m.Chat.Type == "private" || m.Chat.Type == "group" {
+		return ""
+	}
+	if m.Chat.Username != "" {
+		return fmt.Sprintf("https://t.me/%s/%d", m.Chat.Username, m.MessageId)
+	}
+	return strings.Replace(fmt.Sprintf("https://t.me/c/%d/%d", m.Chat.Id, m.MessageId), "-100", "", 1)
```

Comment:
```suggestion
	// Message links use raw chatIds without the -100 prefix; this trims that prefix.
	rawChatId := strings.TrimPrefix(strconv.FormatInt(m.Chat.Id, 10), "-100")
	return fmt.Sprintf("https://t.me/c/%s/%d", rawChatId, m.MessageId)
```

Small nit to ease readability. Too many statements on one line with undefined constants makes the code harder to read.
This updated version should help that and keep things shiny :)